src/CoreBundle/private/modules/CMSDocumentManager/CMSDocumentManager.class.php

```diff
@@ -360,29 +360,29 @@ protected function RenderFileConnectionInfo($aFoundConnections, $sFileId, $bInfo
                         <th>'.TGlobal::Translate('chameleon_system_core.document_manager.connected_item_column_field').'</th>
                     </tr>';
 
-                    $count = 0;
-                    foreach ($aFoundConnections as $tableName => $aFields) {
-                        foreach ($aFields as $field => $value) {
-                            /** @var $oTableRecordList TCMSRecordList */
-                            $oTableRecordList = $value;
-                            /** @var $oRecord TCMSRecord */
-                            while ($oRecord = $oTableRecordList->Next()) {
-                                if ($count % 2) {
-                                    $class = 'evenRowStyleNoHand';
-                                } else {
-                                    $class = 'oddRowStyleNoHand';
-                                }
-
-                                $foundConnectionsHTML .= "<tr class=\"{$class}\">
+            $count = 0;
+            foreach ($aFoundConnections as $tableName => $aFields) {
+                foreach ($aFields as $field => $value) {
+                    /** @var $oTableRecordList TCMSRecordList */
+                    $oTableRecordList = $value;
+                    /** @var $oRecord TCMSRecord */
+                    while ($oRecord = $oTableRecordList->Next()) {
+                        if ($count % 2) {
+                            $class = 'evenRowStyleNoHand';
+                        } else {
+                            $class = 'oddRowStyleNoHand';
```

Comment:
an even-odd rule in css instead?

Reply:
This is not new code... Will not touch this in this PR.